cmd/finch/support_bundle_native.go

```diff
@@ -1,10 +1,10 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-//go:build darwin
+//go:build linux || native
 
 package main
 
-func (nc *nerdctlCommand) GetLimaArgs() []string {
-	return []string{"shell", limaInstanceName, "sudo", "-E"}
+func (gsa *generateSupportBundleAction) assertVMExists() error {
+	return nil
```

Comment:
Do we expect this to be called? If not can we warn here instead of silently returning nil.

Reply:
The bulk of the logic is in the [support_bundle.go](https://github.com/runfinch/finch/pull/1023/files#diff-2f28984dad6126e42d2d40ee664e2266ebc890472b90cd7577b3ecb09bf78f47R73-R88)'s `run` method. I just pulled this function out to avoid splitting that function by platform. Open to better ways to share the code if you have suggestions